wcbench.sh

```diff
@@ -15,7 +15,7 @@ EX_OK=0
 EX_ERR=1
 
 # Output verbose debug info (true) or not (anything else)
-VERBOSE=false
```

Comment:
I suggest not changing this, should default to less-verbose for most folks but support verbose debugging info if needed.